src/Components/Patient/FileUpload.tsx

```diff
@@ -1114,9 +1112,12 @@ export const FileUpload = (props: FileUploadProps) => {
 
   return (
     <div className={hideBack ? "py-2" : "p-4"}>
-      <Modal
-        open={file_state.open}
+      <DialogModal
```

Comment:
Consider using a dedicated `Dialog` component say: `ImagePreviewDialog` with custom implementation for this instead of overriding the CareUI's Dialog component to have bg-transparent and all.